scripts/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM debian:buster
+
+RUN apt-get update
+RUN apt-get install -y git ruby build-essential autoconf bison libssl-dev zlib1g-dev
+RUN apt-get install -y clang libclang-dev curl
+
+RUN git clone https://github.com/ruby/ruby.git ~/clones/ruby
+
+ENV RUBY_VERSION=2_7_2
+
+WORKDIR /root/clones/ruby
+RUN git checkout v${RUBY_VERSION}
+RUN autoconf
+RUN ./configure --disable-install-doc
+RUN make
+RUN make install
+
+RUN git clone https://github.com/rbspy/rbspy.git /opt/rbspy
+
+WORKDIR /opt/rbspy
+ENV PATH=~/clones/ruby:${PATH}
```

Comment:
the only thing i noticed here (which doesn't need to change) is that the ruby directory is `~/clones/ruby` which I guess is because I hardcoded the directory Ruby is in in the script to where it happens to be on my laptop.

Reply:
Yeah, this was to keep it the same as what the script has hard-coded currently. :)